docs/api/migration-guides/qiskit-1.0-features.mdx

```diff
@@ -0,0 +1,600 @@
+  To install `qiskit_aer`, run:
+
+  ```bash
+  pip install qiskit-aer
+  ```
+* The `qiskit.BasicAer` object is no longer available in Qiskit 1.0. See the
+  [`basicaer migration section`](#migrate-from-basicaer-to-basic_provider) for more
+  information on migration options.
+
+* The `qiskit.execute` function is no longer available in Qiskit 1.0.
+  This function served as a high-level wrapper around the transpile and run functionalities
+  in Qiskit. Instead of `qiskit.execute`, you can explicitly use the `transpile` function
+  (with appropriate transpile options) followed by `backend.run()`
+  (with appropriate run options).
+
+  Migration example:
+
+  ```python
+  # Legacy path
+  from qiskit import execute
+  job = execute(circuit, backend)
+
+  # New path
+  from qiskit import transpile
+  new_circuit = transpile(circuit, backend)
+  job = backend.run(new_circuit)
+  ```
+
+  Alternatively, the `Sampler` primitive is semantically equivalent to the
+  deprecated `qiskit.execute` function. The class `BackendSampler`
+  is a generic wrapper for backends that do not support primitives:
+
+  ```python
+  from qiskit.primitives import BackendSampler
+  sampler = BackendSampler(backend)
+  job = sampler.run(circuit)
+  ```
+
+<span id="qiskit.circuit"></span>
+## Changes to `qiskit.circuit`
+
+* The :meth:`.QuantumCircuit.qasm` method used to generate a OpenQASM 2
+  representation of the :class:`.QuantumCircuit` object has been deprecated
+  and will be removed in the Qiskit 1.0.0 release. The
+  :func:`.qasm2.dump` or :func:`.qasm2.dumps` functions which provide similar
+  functionality should be used instead. If you were using the :meth:`.QuantumCircuit.qasm`
+  method to generate pygments formatted output you should instead look at the standalone
+  ``openqasm-pygments`` package to provide this functionality (as :func:`.qasm2.dump`
+  and :func:`.qasm2.dumps` do not provide pygments colored output).
+
+
+<span id="qiskit.converters"></span>
+## Changes to `qiskit.converters`
+
+* The ``qiskit.converters.ast_to_dag`` function has been deprecated and
+  will be removed in the Qiskit 1.0.0 release. It previously was used to
+  convert the abstract syntax tree generated by the legacy OpenQASM 2 parser
+  (in the :mod:`qiskit.qasm` module which has been deprecated) and convert
+  that directly to a :class:`.DAGCircuit`. As the legacy OpenQASM 2 parser
+  has been deprecated this function will no longer serves a purpose after the
+  legacy parser is removed. If you were previously using this, you can
+  instead parse your OpenQASM 2 files into a :class:`.QuantumCircuit` using
+  the :meth:`.QuantumCircuit.from_qasm_file` or
+  :meth:`.QuantumCircuit.from_qasm_str` constructor methods and then
+  converting that :class:`.QuantumCircuit` into a :class:`.DAGCircuit`
+  with :func:`.circuit_to_dag`.
+
+<span id="qiskit.extensions"></span>
+## Changes to `qiskit.extensions`
```

Comment:
This also misses the removal of

    * ``SingleQubitUnitary`` (instead use :class:`.library.UnitaryGate`),
    * ``Snapshot`` (superseded by Qiskit Aer's save instructions),
    * ``ExtensionError``,